modules/frontend/search_progress.go

```diff
@@ -159,6 +159,17 @@ func (r *searchProgress) result() *shardedSearchResults {
 			SpanSet:           copySpanset(m.SpanSet),
 		}
 
+		// copy service stats
+		if len(m.ServiceStats) > 0 {
```

Comment:
i do, admittedly, wish we had separated the parquet changes from the search pipeline changes, but i don't think it's worth reverting at this point.

cc @grafana/observability-traces-and-profiling search results will be enriched with services, span and error counts when we move to vparquet4. nothing actionable now, but something we can add in the future.